src/tools/PICS-generator/PICSGenerator.py

```diff
@@ -134,7 +138,8 @@ def GenerateDevicePicsXmlFiles(clusterName, clusterPicsCode, featurePicsList, at
     # Attributes PICS
     # TODO: Only check if list is not empty
     # console.print(attributePicsList)
-    serverAttributesNode = root.find("./clusterSide[@type='Server']/attributes")
+    serverAttributesNode = root.find(
```

Comment:
please change your autoformat plugin settings - this PR seems to need to be 2-3 line change, instead it got to 43/20 ... restyle.io should be the only restyler.

Reply:
This is from restyler

Reply:
I wondered about this as well. but I figured we updated the restyler since I did the original version of this?